### DISCUSSION TITLE
Hi where is diablo.ini pls

### DISCUSSION THREAD

**Discussion by ponchikkkk** [diasurgical/DevilutionX#4028] [General]
sorry if i sound like an idiot but .... i bought the game from GOG ...  installed devilutionX ....

and found this page   https://github.com/diasurgical/devilutionX/wiki/DevilutionX-diablo.ini-configuration-guide 

but i cant find the diablo.ini to edit it ???

it is not in the install folder ??

thx in advance

**Comment by AJenbo** [comment]
My advice is to let us know if you use Android, Windows, Linux, Mac or something else?

**Reply by ponchikkkk** [reply]
thx anyway

windows but i actually found it now 
C:\Users\.....your user.......\AppData\Roaming\diasurgical\devilution

if anyone needs it  
didnt look for it in roaming before  and windows search definately doesnt find it

**Reply by AJenbo** [reply]
Ok, this is actually mentioned in the readme included with the download:
```
# Save Games and configurations
The configurations and save games are located in:
C:\Users\[username]\AppData\Roaming\diasurgical\devilution
```

You should also be able enter the `%AppData%\diasurgical\devilution` as mentioned in the wiki page you listed, in explorer to go directly to the folder.
